.flowconfig

```diff
@@ -4,6 +4,7 @@
 [ignore]
 <PROJECT_ROOT>/packages/gestalt-codemods/.*/__testfixtures__
 <PROJECT_ROOT>/cypress
+<PROJECT_ROOT>/node_modules/cypress/
```

Comment:
Added this since Cypress does not expose flow types but the VSCode extension keeps crashing on it.